gettsim/parameters/arbeitsl_geld_2.yaml

```diff
@@ -282,42 +284,77 @@ mehrbedarf_anteil:
     max: 0.6
     reference: G. v. 24.12.2003 BGBl. I S. 2954.
 
-vermögensfreibetrag:
+
+vermögensfreibetrag_obergrenze:
   name:
-    de_1: Vermögensgrundfreibetrag je Lebensjahr, nach Geburtsjahr, erst ab 2008 unterschiedlich
-    de_2: Vermögensfreibetrag für Erstausstattung
-    de_3: Vermögensgrundfreibetrag je Kind
-    en_1: Basic allowance per year of life, sorted by year of birth, starting 2008 different
-    en_2: Allowance for furniture
-    en_3: Allowance per child
+    de: Maximaler Vermögensgrundfreibetrag je Lebensjahr. Ab 2008 nach Geburtsjahr unterschiedlich
+    en: Basic allowance per year of life. Sorted by year of birth starting 2008 different
   description:
     de: § 12 (2) Satz 1 Nr. 1 SGB II.
     en:
   unit: Euro
   2005-01-01:
-    standard: 200
-    vor_1948: 520
-    1948_bis_1957: 13000
-    1958_bis_1963: 13000
-    nach_1963: 13000
-    ausstattung: 750
-    kind: 4100
+    scalar: 13000
     reference: Artikel 1 G. v. 24.12.2003 BGBl. I S. 2954.
   2006-08-01:
-    deviation_from: previous
-    standard: 150
-    1948_bis_1957: 9750
-    1958_bis_1963: 9750
-    nach_1963: 9750
-    kind: 3100
+    scalar: 9750
     reference: Artikel 1 G. v. 20.07.2006 BGBl. I S. 1706.
   2008-01-01:
-    deviation_from: previous
-    1948_bis_1957: 9750
+    vor_1957: 9750
     1958_bis_1963: 9900
     nach_1963: 10050
     note: Artikel 1 G. v. 20.04.2007 BGBl. I S. 554.
 
+
+
+vermögensfreibetrag:
+  name:
+    de: Vermögensgrundfreibetrag je Lebensjahr, gestaffelt nach Geburtsjahr
+    en: Basic allowance per year of life, sorted by year of birth
+  description:
+    de: § 12 (2) Satz 1 Nr. 1 SGB II.
+    en:
+  unit: Euro
+  2005-01-01:
+    ab_1948: 200
```

Comment:
Maybe more verbose `geboren_vor_1948`, `geboren_zw_1958_und_1963` etc. ?

Reply:
What do you thing of geb_vor_1949 etc.?

Reply:
`_bis_[t]`, `_ab_[t+1]_bis_[s]`, `ab_[s+1]` -- should be reasonably clear that it always includes the years mentioned and the keywords work regardless of whether interval is closed or open-ended.

Reply:
`geb_bis_1948` `geb_ab_1949` sounds good (not sure if this is correct or it should be bis_1947, ab_1948)

Reply:
btw., do you know where `vor_1948: 520` is specified in the law?

I didn't find a differentiation by birth year in § 12 (2) Satz 1 Nr. 1 SGB II.

Reply:
I can't find the differentiation either. Should we delete it?

Reply:
@Eric-Sommer, do you have an idea where this comes from? If not, let's delete it.

Reply:
Maybe here: https://www.buzer.de/gesetz/2602/a37356.htm

> (5) § 12 Abs. 2 Nr. 1 gilt mit der Maßgabe, dass für die in § 4 Abs. 2 Satz 2 der Arbeitslosenhilfe-Verordnung vom 13. Dezember 2001 (BGBl. I S. 3734) in der Fassung vom 31. Dezember 2004 genannten Personen an die Stelle des Grundfreibetrags in Höhe von 150 Euro je vollendetem Lebensjahr ein Freibetrag von 520 Euro, an die Stelle des Höchstfreibetrags in Höhe von jeweils 9.750 Euro ein Höchstfreibetrag in Höhe von 33.800 Euro tritt.

Reply:
Thank you.

This seems to be the reference "Arbeitslosenhilfe-Verordnung vom 13. Dezember 2001"

https://www.bgbl.de/xaver/bgbl/start.xav?start=%2F%2F*%5B%40attr_id%3D%27bgbl101s3734.pdf%27%5D#__bgbl__%2F%2F*%5B%40attr_id%3D%27bgbl101s3734.pdf%27%5D__1643816721703

But it is about the conversion from DM to EUR. Do you see any reference to "born before 1948" as indicated by the parameter?

Reply:
I cannot find the version from 31.12.2004. But [this](http://www.portal-sozialpolitik.de/uploads/sopo/pdf/2002/2002-11-05-Hartz-I-II-03-02-00-ALHI-Steffen.pdf) is consistent.

However, it would be "bis_1947" and "ab_1948".

Please change and merge this one @ChristianZimpelmann @MaxBlesch -- no need to spend ages on a tiny historical detail.